src/igvfd/types/sample.py

```diff
@@ -130,3 +130,15 @@ class DifferentiatedCell(Biosample):
 class TechnicalSample(Sample):
     item_type = 'technical_sample'
     schema = load_schema('igvfd:schemas/technical_sample.json')
+
+
+@collection(
+    name='whole-organisms',
+    unique_key='accession',
+    properties={
+        'title': 'Whole Organism Samples',
```

Comment:
I think the title should be "Whole organism samples" 
I don't think each word should be capitalized

Reply:
Done